phpunit/class-wp-rest-pattern-directory-controller-test.php

```diff
@@ -212,23 +212,74 @@ function ( $preempt, $args, $url ) {
 	}
 
 	/**
-	 * @doesNotPerformAssertions
+	 * @covers WP_REST_Pattern_Directory_Controller::prepare_item_for_response
+	 *
+	 * @since 5.8.0
+	 * @since 6.2.0 Added `block_types` property.
 	 */
-	public function test_context_param() {
-		// Covered by the core test.
+	public function test_prepare_item() {
+		$raw_patterns                 = json_decode( self::get_raw_response( 'browse-all' ) );
+		$raw_patterns[0]->extra_field = 'this should be removed';
+
+		$prepared_pattern = static::$controller->prepare_response_for_collection(
+			static::$controller->prepare_item_for_response( $raw_patterns[0], new WP_REST_Request() )
+		);
+
+		$this->assertPatternMatchesSchema( $prepared_pattern );
+		$this->assertArrayNotHasKey( 'extra_field', $prepared_pattern );
+	}
+
+	/**
+	 * Asserts that the pattern matches the expected response schema.
+	 *
+	 * @param WP_REST_Response[] $pattern An individual pattern from the REST API response.
+	 */
+	public function assertPatternMatchesSchema( $pattern ) {
```

Comment:
I think we already have this in core tests 👍

Reply:
Yes, but since we are updating the schema, I think it's good to have them here.. This will not be copied to core during the back port.